lib/src/messages.dart

```diff
@@ -11,8 +11,8 @@ class _MsgDict {
   _MsgDict._({
     this.dartanalyzerShortName = 'dart analyze',
     this.dartanalyzerShortCmd = 'dart analyze .',
-    this.dartfmtShortName = 'dartfmt',
-    this.dartfmtShortCmd = 'dartfmt -n .',
+    this.dartfmtShortName = 'dart format',
```

Comment:
I have a feeling these are unused. @isoos can these be removed?

Reply:
I think they are used? 

https://github.com/dart-lang/pana/search?q=runningDartanalyzerFailed

Reply:
https://github.com/dart-lang/pana/search?q=dartfmtShortName shows no usage sites...

Reply:
Ah, sorry, I misunderstood. Took at stab at updating the PR. Let's see if that passes CI...